axum-extra/src/lib.rs

```diff
@@ -83,6 +83,9 @@ pub mod routing;
 #[cfg(feature = "json-lines")]
 pub mod json_lines;
 
+#[cfg(feature = "multipart")]
+pub mod multipart_builder;
```

Comment:
Why "builder"?

Reply:
There's already a `Multipart` extractor, so I wanted to clarify between that and the functionality for creating forms.

Reply:
That's definitely an improvement IMO, I felt multipart_builder was verbose but didn't know what else to do